hrms/hr/doctype/appraisal_cycle/appraisal_cycle.js

```diff
@@ -3,6 +3,44 @@
 
 frappe.ui.form.on("Appraisal Cycle", {
 	refresh(frm) {
+		async function set_autocompletions_for_final_score_formula(frm) {
```

Comment:
move this function outside refresh after `show_custom_buttons`. A closure is not really needed here

Reply:
resolved